[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_filter_get_message_after_node1_restarts with id: 2024-12-04_11-44-47__487e7e02-8bc9-4578-9dec-07540ccb1378
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.filter:filter.py:28 Running fixture setup: filter_setup
[35mDEBUG   [0m src.steps.filter:filter.py:34 Running fixture setup: setup_main_relay_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-12-04_11-44-47__487e7e02-8bc9-4578-9dec-07540ccb1378__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.168.119
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['37224', '37225', '37226', '37227', '37228']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 37224:37224 -p 37225:37225 -p 37226:37226 -p 37227:37227 -p 37228:37228 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=37226 --rest-port=37224 --tcp-port=37225 --discv5-udp-port=37227 --rest-address=0.0.0.0 --nat=extip:172.18.168.119 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=3141e5fac2630c306ddd4ab15fca5eb1c5388bafc9664eb5c133a604fc4a226a --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=37228 --metrics-logging=true --relay=true --filter=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.168.119 waku 3654da03b14068b898947dfe6bc082b605e7b69fb34ed0b2fe0847968f843b28
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 3654da03b140. Setting up logs at ./log/docker/node1_2024-12-04_11-44-47__487e7e02-8bc9-4578-9dec-07540ccb1378__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 37224
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37224/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37224/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.168.119/tcp/37225/p2p/16Uiu2HAmVp6d2jU8mxiPCBrVRFQ53643RdMXBFt9XrtE7MuPoKU2","/ip4/172.18.168.119/tcp/37226/ws/p2p/16Uiu2HAmVp6d2jU8mxiPCBrVRFQ53643RdMXBFt9XrtE7MuPoKU2"],"enrUri":"enr:-L24QM3pYWAFkDkGyAdGfIdP2ebahT6ey4j5Jk8Go8-MfQ9zBf5GDPZGtXvZApuLE_vbc4NfvbbCVrYHwAGc5Qzn5DACgmlkgnY0gmlwhKwSqHeKbXVsdGlhZGRyc5YACASsEqh3BpFpAAoErBKodwaRat0DgnJzhQADAQAAiXNlY3AyNTZrMaED_uv_AX7dHCJjwzEJLt9nfDaeagNJkuB8rX3xZsAoFLeDdGNwgpFpg3VkcIKRa4V3YWt1MgU"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[35mDEBUG   [0m src.steps.filter:filter.py:39 Running fixture setup: setup_main_filter_node
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node2_2024-12-04_11-44-47__487e7e02-8bc9-4578-9dec-07540ccb1378__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.110.165
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['17893', '17894', '17895', '17896', '17897']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:439 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:168 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:170 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 17893:17893 -p 17894:17894 -p 17895:17895 -p 17896:17896 -p 17897:17897 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=17895 --rest-port=17893 --tcp-port=17894 --discv5-udp-port=17896 --rest-address=0.0.0.0 --nat=extip:172.18.110.165 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=dff15a4cb12e651beecce481e14963c34aed368dca6d41c9421c8d8fa97d178d --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=17897 --metrics-logging=true --relay=false --discv5-bootstrap-node=enr:-L24QM3pYWAFkDkGyAdGfIdP2ebahT6ey4j5Jk8Go8-MfQ9zBf5GDPZGtXvZApuLE_vbc4NfvbbCVrYHwAGc5Qzn5DACgmlkgnY0gmlwhKwSqHeKbXVsdGlhZGRyc5YACASsEqh3BpFpAAoErBKodwaRat0DgnJzhQADAQAAiXNlY3AyNTZrMaED_uv_AX7dHCJjwzEJLt9nfDaeagNJkuB8rX3xZsAoFLeDdGNwgpFpg3VkcIKRa4V3YWt1MgU --filternode=/ip4/172.18.168.119/tcp/37225/p2p/16Uiu2HAmVp6d2jU8mxiPCBrVRFQ53643RdMXBFt9XrtE7MuPoKU2
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.110.165 waku fe7e154537a21105cb40bcd76c79effb325a872cc905772dd84858d421187273
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID fe7e154537a2. Setting up logs at ./log/docker/node2_2024-12-04_11-44-47__487e7e02-8bc9-4578-9dec-07540ccb1378__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:182 Started container from image wakuorg/nwaku:latest. REST: 17893
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:17893/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:17893/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.110.165/tcp/17894/p2p/16Uiu2HAmUfjrQjm6ZV5i1ZbXcsAkr2PAgyYEnPTM5aW3E36eLXJj","/ip4/172.18.110.165/tcp/17895/ws/p2p/16Uiu2HAmUfjrQjm6ZV5i1ZbXcsAkr2PAgyYEnPTM5aW3E36eLXJj"],"enrUri":"enr:-L24QJsh2N2YRUOATaDT6V0pjwe35RcrOG3RDeVtipRwS5VnVxncYcRfzgv0RTEseTVX6mZuVmh4PKx5y585cho0bFUCgmlkgnY0gmlwhKwSbqWKbXVsdGlhZGRyc5YACASsEm6lBkXmAAoErBJupQZF590DgnJzhQADAQAAiXNlY3AyNTZrMaED7ewtw_z7x3Na4yEB0xWt6GpjM4bF19darlJxM8nge6SDdGNwgkXmg3VkcIJF6IV3YWt1MgA"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:17893/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.168.119/tcp/37225/p2p/16Uiu2HAmVp6d2jU8mxiPCBrVRFQ53643RdMXBFt9XrtE7MuPoKU2"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.filter:filter.py:47 Running fixture setup: subscribe_main_nodes
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:37224/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:17893/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "5c93ff13-446e-48a7-b780-dde67a6429fc", "contentFilters": ["/test/1/waku-filter/proto"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"5c93ff13-446e-48a7-b780-dde67a6429fc","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:37224/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "/test/1/waku-filter/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:98 Checking that peer NODE_2:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:17893/filter/v2/messages/%2Ftest%2F1%2Fwaku-filter%2Fproto" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"RmlsdGVyIHdvcmtzISE=","contentTopic":"/test/1/waku-filter/proto","version":0,"timestamp":1733312707201108224,"ephemeral":false}]'
[35mDEBUG   [0m src.node.waku_node:waku_node.py:249 Restarting container with id 3654da03b140
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37224/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37224/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37224/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.168.119/tcp/37225/p2p/16Uiu2HAmVp6d2jU8mxiPCBrVRFQ53643RdMXBFt9XrtE7MuPoKU2","/ip4/172.18.168.119/tcp/37226/ws/p2p/16Uiu2HAmVp6d2jU8mxiPCBrVRFQ53643RdMXBFt9XrtE7MuPoKU2"],"enrUri":"enr:-L24QM3pYWAFkDkGyAdGfIdP2ebahT6ey4j5Jk8Go8-MfQ9zBf5GDPZGtXvZApuLE_vbc4NfvbbCVrYHwAGc5Qzn5DACgmlkgnY0gmlwhKwSqHeKbXVsdGlhZGRyc5YACASsEqh3BpFpAAoErBKodwaRat0DgnJzhQADAQAAiXNlY3AyNTZrMaED_uv_AX7dHCJjwzEJLt9nfDaeagNJkuB8rX3xZsAoFLeDdGNwgpFpg3VkcIKRa4V3YWt1MgU"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:287 REST service is ready !!
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 2 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:37224/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:17893/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "ff0f9022-ca69-4612-b586-4fa19f031a16", "contentFilters": ["/test/1/waku-filter/proto"], "pubsubTopic": "/waku/2/rs/3/1"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"ff0f9022-ca69-4612-b586-4fa19f031a16","statusDesc":"OK"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:37224/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "RmlsdGVyIHdvcmtzISE=", "contentTopic": "/test/1/waku-filter/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.filter:filter.py:98 Checking that peer NODE_2:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:17893/filter/v2/messages/%2Ftest%2F1%2Fwaku-filter%2Fproto" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"RmlsdGVyIHdvcmtzISE=","contentTopic":"/test/1/waku-filter/proto","version":0,"timestamp":1733312742859341056,"ephemeral":false}]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id 3654da03b140
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:226 Stopping container with id fe7e154537a2
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 3654da03b140. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:233 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.